modules/parsers/parser-hdbdd/src/main/java/com/sap/xsk/parser/hdbdd/custom/ArtifactDefinitionListener.java

```diff
@@ -499,7 +500,7 @@ public void enterSelectRule(SelectRuleContext ctx) {
     }
 
     if (ctx.dependsOnTable != null) {
-      selectSymbol.setDependsOnTable(handleStringLiteral(ctx.dependsOnTable.getText()));
+      selectSymbol.setDependsOnTable(handleStringLiteral(ctx.dependsOnTable.stream().map(RuleContext::getText).collect(Collectors.joining("."))));
```

Comment:
Move the stream operation and `handleStringLiteral` into separate variables. Also, what does `handleStringLiteral` do?

Reply:
I guess it removes quotes and escaped quotes in strings.

Reply:
Could you rename it to something more meaningful if it removes quotes?

Reply:
This wasn't implemented by me, I can't think of a more semantically appropriate name without making it sound weird